packages/app/src/systems/Bridge/components/BridgeSteps/BridgeSteps.tsx

```diff
@@ -87,7 +87,7 @@ const styles = {
     color: '$intentsBase12',
   }),
   icon: cssObj({
-    color: '$black12',
+    color: '$intentsBase12 !important',
```

Comment:
I changed for black before because was looking bad in dark theme.. still is, I think we should force as black here

![image](https://github.com/FuelLabs/fuels-portal/assets/8636507/6664ab14-c684-4a57-9eac-23b810759c9b)

Reply:
or adapt the green to make it look good in both themes